apps/src/netsim/NetSimShard.js

```diff
@@ -119,6 +128,20 @@ var NetSimShard = module.exports = function (shardID, pubSubConfig) {
 };
 
 /**
+ * Necessary tear-down for shard.  In particular, disconnecting
+ * from pubsub service.
+ */
+NetSimShard.prototype.disconnect = function () {
+  this.nodeTable.unsubscribe();
+  this.wireTable.unsubscribe();
+  this.messageTable.unsubscribe();
+  this.logTable.unsubscribe();
+  this.pubSubChannel.unsubscribe(ALL_TABLES);
```

Comment:
It would be nice to making the naming a little more clear here; I could easily see there being some future confusion about the `ALL_TABLES` subscription being for events that span all the tables as opposed to all the events for all the tables.